packages/components/src/composite/Tabs/Container.tsx

```
@@ -0,0 +1,238 @@
+        resizeObserverRef.current.disconnect();
+      }
+      const height =
+        scrollTabElementsRef.current?.[focusedTab.value]?.element.clientHeight;
+      if (height) {
+        (
+          listContainerRef.current as HTMLElement
+        ).style.maxHeight = `${height}px`;
+        setTimeout(() => {
+          resizeObserverRef.current = new ResizeObserver((entries) => {
+            const entry = entries[0];
+            if (entry && entry.contentRect.height) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)